mne/io/base.py

```diff
@@ -2409,10 +2410,13 @@ def concatenate_raws(raws, preload=None, events_list=None, verbose=None):
     Parameters
     ----------
     raws : list
-        List of Raw instances to concatenate (in order).
+        List of `~mne.io.Raw` instances to concatenate (in order).
     %(preload_concatenate)s
     events_list : None | list
-        The events to concatenate. Defaults to None.
+        The events to concatenate. Defaults to ``None``.
+    on_mismatch : 'raise' | 'warn' | 'ignore'
+        How to handle situations where the device-to-head transformation
+        differs between raw instances. Defaults to ``'raise'``.
```

Comment:
Should make use of `_on_missing_base` and docdict, see `mne/utils/docs.py`

Also needs `.. versionadded` directive

Reply:
Argh... yes to all. Thanks for taking your time